README.md

```diff
@@ -977,17 +980,17 @@ If reporting a neutral killer, Amnesiac becomes that neutral killer.<br>
 
 ### Egoist
 
-Create by そうくん<br>
+Created by そうくん<br>
 Original idea by しゅー<br>
 
 Team : Neutral<br>
 Basis : Shapeshifter<br>
-Victory Condition : Satisfy the Impostor victory condition after all the Impostors die.<br>
+Victory Condition : Feel the Impostor victory after all the Impostors die.<br>
```

Comment:
The original sentence had clear intentions in English. Your sentence doesn't really make sense.
If you still do not like the original sentence, here is an altered version:
"Victory Condition: Win the Impostor victory once all the Imposters are gone."